mypyc/test-data/irbuild-tuple.test

```diff
@@ -73,24 +73,18 @@ L0:
     return r2
 
 [case testSequenceTupleForced]
-from typing import Tuple
-def f() -> int:
-    t = (1, 2) # type: Tuple[int, ...]
+from typing import Tuple, cast
+def f(t: Tuple[int, ...]) -> int:
     return t[1]
 [out]
-def f():
-    r0 :: tuple[int, int]
-    r1 :: object
+def f(t):
     t :: tuple
```

Comment:
what is this syntax

Reply:
erm, I think it's a representation of IR or something.

Reply:
What's ir